packages/footer-component/src/components/dc-footer/dc-footer.test.ts

```diff
@@ -0,0 +1,107 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { DcFooter } from './dc-footer';
+
+describe('dc-footer', () => {
+  it.only('renders', async () => {
+    const { root } = await newSpecPage({
+      components: [DcFooter],
+      html: '<dc-footer></dc-footer>',
+    });
+
+    expect(root).toMatchInlineSnapshot(`
```

Comment:
I will rather send such a long snapshot to a separate file _(avoid the "InlineSnapshot" here)_ but cool at least we are getting into some attempt of testing 👌

Reply:
Let's discuss this in slack, I'll merge this